app/models/history.rb

```diff
@@ -0,0 +1,29 @@
+class History < FlexiblePage
+private
+
+  def default_flexible_sections
+    [
+      {
+        type: "page_title",
+        context: "History",
+        heading_text: title,
+        lead_paragraph: content_store_response.dig("details", "lead_paragraph"),
+      },
+      {
+        type: "rich_content",
+        contents_list: (content_store_response.dig("details", "headers") || []).map { |header| header.except("headers") },
+        govspeak: body,
+        image: image_hash,
+      },
+    ]
+  end
+
+  def image_hash
+    return nil unless content_store_response.dig("details", "sidebar_image")
+
+    {
+      alt: content_store_response.dig("details", "sidebar_image", "caption"),
+      src: content_store_response.dig("details", "sidebar_image", "url"),
+    }
+  end
```

Comment:
Is it more efficient to store  the `content_store_response.dig("details", "sidebar_image")` part in a variable here so that it's only called once?